Cargo.lock

```diff
@@ -1978,6 +1999,19 @@ dependencies = [
  "digest",
 ]
 
+[[package]]
+name = "mdns-sd"
+version = "0.10.4"
+source = "registry+https://github.com/rust-lang/crates.io-index"
+checksum = "1961985d2b8f6764565cfe77916f7926aabb6b377cfb5ddf191f7e34136b547f"
+dependencies = [
+ "flume",
+ "if-addrs",
+ "log",
+ "polling 2.8.0",
+ "socket2 0.4.10",
```

Comment:
issue (non-blocking): This crate is used outdated dependencies, and this causes dependency duplication. Do you think you could send a PR for updating them soonish? It’s okay to wait until you are done with implementing the feature first. If you don’t do it now, can you create a Jira ticket to not forget? Thank you.

Reply:
https://github.com/keepsimple1/mdns-sd/pull/172

Reply:
created PR to mdsn-sd